pkg/services/gitops/install.go

```diff
@@ -26,13 +31,11 @@ type InstallParams struct {
 
 func (g *Gitops) Install(params InstallParams) (map[string][]byte, error) {
 	ctx := context.Background()
-	status := g.kube.GetClusterStatus(ctx)
 
-	switch status {
-	case kube.FluxInstalled:
-		return nil, errors.New("Weave GitOps does not yet support installation onto a cluster that is using Flux.\nPlease uninstall flux before proceeding:\n  $ flux uninstall")
-	case kube.Unknown:
-		return nil, errors.New("Weave GitOps cannot talk to the cluster")
+	g.logger.Actionf("Validating Weave Gitops installation")
```

Comment:
This Log message here, breaks the output of validateWegoInstall.

https://github.com/weaveworks/aws-marketplace/runs/4500418327?check_suite_focus=true

Results in a generated output in case dry run is used such as this:

```
► Validating Weave Gitops installation
---
# Flux version: v0.21.0
# Components: source-controller,kustomize-controller,helm-controller,notification-controller,image-reflector-controller,image-automation-controller
apiVersion: v1
kind: Namespace
metadata:
  labels:
    app.kubernetes.io/instance: wego-system
    app.kubernetes.io/part-of: flux
    app.kubernetes.io/version: v0.21.0
  name: wego-system
---
```